<launch>
	<node name="recognizer_2" pkg="pocketsphinx" type="recognizer.py" output="screen">
		<param name="mic_name" value="alsa_input.usb-Logitech_Logitech_G35_Headset-00-Headset_1.analog-mono"/>
		<param name="lm" value="$(find pocketsphinx)/demo/robocup.lm"/>
		<param name="dict" value="$(find pocketsphinx)/demo/robocup.dic"/>
	</node>
</launch>
